server/configmodule.py

```diff
@@ -102,6 +103,8 @@ class LocalConfig(Config):
     SECRET_PROJECT = 'datcom-website-dev'
     API_ROOT = 'https://autopush.api.datacommons.org'
     RECON_API_ROOT = 'https://autopush.recon.datacommons.org'
+    AI_CONFIG_PATH = os.path.abspath(
```

Comment:
You can mount it in the GKE. An example is redis.json https://github.com/datacommonsorg/website/blob/master/deploy/overlays/prod/patch_deployment.yaml#L38. Then you don't have do this for each environment as below

Reply:
Got it, that makes sense.